src/components/CompleteOptionsInverted.tsx

```diff
@@ -89,7 +89,6 @@ export function CompleteOptionsInverted({
           {orderedSelections.map(([purpose, isChecked]) => (
             <Toggle
               key={purpose}
-              invertLabels
```

Comment:
is this a css change?

Reply:
I was just being safe by wholly reverting all commits after 2.10.3

Reply:
ohhh i see. i think i was confused about what you were trying to do (reverting just ben's PR vs reverting everything)

Reply:
we can do a diff between 2.10.3 and 2.10.8 to see if any other css has changed...?